### DISCUSSION TITLE
Does Quiver's license permit me to modify it and create a closed-source version?

### DISCUSSION THREAD

**Discussion by enjoysmath** [varkor/quiver#91] [Q&A] [answered]
I've ceased development on QuiverDatabase for technical difficulty reasons, but since I got diagram search working (with variable replacement), and I want to have content on my next site searchable using a diagram, I will be re-using some of that code.

Was wondering how permissive is the MIT license?  

Regardless of the license, what would like me to show on the final product?  For example, a link to the standalone editor q.uiver.app or two this github repo.

I will have a start-up fee per user account of $1-5 and charge the users on a per-search basis (they purchase a quantity of search units), in order to cover the cost of Graphene Neo4j hosting (the graph content will 1-to-1 correspond with objects / arrows in a Neo4j graph database like they currently do).

The main money though is not just for the site, but is made in a marketplace created by the user community.  There are several major online flashcard companies such as Quizlet and Brainscape, but either they don't support LaTeX or they don't offer this marketplace to sell your cards on.  

Creating flashcards for yourself can be time-consuming.  And so naturally, you'd wish to sell your end result to future students.

Not only will the users be able to create flashcards, but through NLP the answer a user types in can be checked against the right answer.  Diagram user input might be "fill in the missing arrows / objects and label them".  

So not complex games (like Duo Lingo has), but just simple flashcard-type questions / answers.  The users can say teach a proof of the Snake Lemma from Homological Algebra.  In that case the cards will have a natural topological ordering instead of a randomized as in a regular flashcard session.

There will be one diagram allowed for the fronts of cards and one diagram allowed for the backs of cards.  Max number of nodes in a single diagram is 50 for an approximately 7x7 grid and max number of edges is then of course 100.

That is as full of a description of the final product as I can give, so please let me know what I can do so that I can close my source, if that's even allowed...

**Comment by varkor** [comment, marked as answer]
The MIT licence [permits commercial usage](https://choosealicense.com/licenses/mit/). It is only necessary to include a copy of the original copyright notice, though it would be nice to include a link to **quiver** somewhere if you were happy to :)
